assets/home/sections/get-help.js

```diff
@@ -1,22 +1,110 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { Icon, help, lock, external } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import Section from '../section';
 
+/**
+ * Component representing each of the items under a Help Category.
+ *
+ * @param {Object}      props           Component properties.
+ * @param {string}      props.title     The title.
+ * @param {string|null} props.url       Optional url. When missing the title will not be clickable.
+ * @param {Object|null} props.extraLink An extra link that will be displayed next to the main title.
+ */
+const Item = ( { title, url, extraLink } ) => {
+	const isTitleInteractive = url !== null;
+
+	const link = isTitleInteractive ? (
+		<a href={ url } target="_blank" rel="noreferrer">
+			{ title }
+		</a>
+	) : (
+		title
+	);
+
+	return (
+		<li className="sensei-home__help-item">
+			<div className="sensei-home__help-item__icon">
```

Comment:
I think we need to handle the styles a little different here to the link be aligned with the text when it breaks the line.

<img width="171" alt="Screen Shot 2022-10-11 at 11 34 46" src="https://user-images.githubusercontent.com/876340/195122802-ac58caf9-1551-4e9a-a2eb-384a76a740c5.png">

Reply:
Fixed here: bc0765787